src/style.css

```diff
@@ -1,3 +1,60 @@
 body {
+  font-family: Roboto, "Times New Roman", sans-serif;
+  margin: 0 50px;
+  --link-color: #00acdc;
```

Comment:
this color is for the active link and the hover state
it's not just the link color
so will be better rename